src/uia.cpp

```diff
@@ -192,8 +194,8 @@ bool terminateUia() {
 	if (uiaProvider) {
 		// Null out uiaProvider so it can't be returned by WM_GETOBJECT during
 		// disconnection.
-		CComPtr<IRawElementProviderSimple> tmpProv = move(uiaProvider);
-		uiaProvider = nullptr;
+		CComPtr<IRawElementProviderSimple> tmpProv = uiaProvider;
+		uiaProvider.Release(); // contained pointer  is now null.
```

Comment:
I'm slightly worried about the ordering here. Should we first release, than call DisconnectProvider or vise versa?